Formula/jsign.rb

```diff
@@ -0,0 +1,50 @@
+class Jsign < Formula
+  desc "Tool for signing Windows executable files, installers and scripts"
+  homepage "https://ebourg.github.io/jsign/"
+  url "https://github.com/ebourg/jsign/archive/refs/tags/5.0.tar.gz"
+  sha256 "7b77a12aaea4f404e7b243bd58cfde485eb03b44219e128338c9fe6617ad1fa1"
+  license "Apache-2.0"
+  head "https://github.com/ebourg/jsign.git", branch: "master"
+
+  depends_on "maven" => :build
+  depends_on "openjdk@17" # The build fails with more recent JDKs
+
+  def install
+    ENV["JAVA_HOME"] = Formula["openjdk@17"].opt_prefix
+    system("mvn", "--batch-mode", "package", "--projects", "jsign-core,jsign-cli,jsign-ant,jsign", "-DskipTests",
+           "-Djdeb.skip", "-Dmaven.javadoc.skip")
```

Comment:
Let's make this more readable:
```suggestion
    system "mvn", "--batch-mode", "package",
                  "--projects", "jsign-core,jsign-cli,jsign-ant,jsign",
                  "-DskipTests",
                  "-Djdeb.skip",
                  "-Dmaven.javadoc.skip"
```

Reply:
Applied